_includes/manuals/2.0/5.1.12_dynflow_scaling.md

```diff
@@ -0,0 +1,172 @@
+Created symlink /etc/systemd/system/multi-user.target.wants/dynflow-sidekiq@worker-2.service → /lib/systemd/system/dynflow-sidekiq@.service.
+
+root@modest-gator:/etc/foreman/dynflow# systemctl status dynflow-sidekiq@worker-{1,2}
+● dynflow-sidekiq@worker-1.service - Foreman jobs daemon - worker-1 on sidekiq
+   Loaded: loaded (/lib/systemd/system/dynflow-sidekiq@.service; enabled; vendor preset: enabled)
+   Active: active (running) since Thu 2020-05-14 11:24:19 UTC; 6s ago
+     Docs: https://theforeman.org
+ Main PID: 1422 (ruby2.5)
+    Tasks: 5 (limit: 38437)
+   Memory: 175.9M
+   CGroup: /system.slice/system-dynflow\x2dsidekiq.slice/dynflow-sidekiq@worker-1.service
+           └─1422 ruby2.5 /usr/share/foreman/vendor/ruby/2.5.0/bin/sidekiq -e production -r /usr/share/foreman/extras/dynflow-sidekiq.rb -C /etc/foreman/dynflow/worker-1.yml
+
+May 14 11:24:19 modest-gator systemd[1]: Started Foreman jobs daemon - worker-1 on sidekiq.
+May 14 11:24:21 modest-gator dynflow-sidekiq@worker-1[1422]: 2020-05-14T11:24:21.442Z 1422 TID-gpznscy7m INFO: GitLab reliable fetch activated!
+May 14 11:24:21 modest-gator dynflow-sidekiq@worker-1[1422]: 2020-05-14T11:24:21.443Z 1422 TID-gpzo69up6 INFO: Booting Sidekiq 5.2.8 with redis options {:id=>"Sidekiq-server-PID-1422", :url=>"redis://localhost:6379/0"}
+
+● dynflow-sidekiq@worker-2.service - Foreman jobs daemon - worker-2 on sidekiq
+   Loaded: loaded (/lib/systemd/system/dynflow-sidekiq@.service; enabled; vendor preset: enabled)
+   Active: active (running) since Thu 2020-05-14 11:24:19 UTC; 6s ago
+     Docs: https://theforeman.org
+ Main PID: 1423 (ruby2.5)
+    Tasks: 5 (limit: 38437)
+   Memory: 178.4M
+   CGroup: /system.slice/system-dynflow\x2dsidekiq.slice/dynflow-sidekiq@worker-2.service
+           └─1423 ruby2.5 /usr/share/foreman/vendor/ruby/2.5.0/bin/sidekiq -e production -r /usr/share/foreman/extras/dynflow-sidekiq.rb -C /etc/foreman/dynflow/worker-2.yml
+
+May 14 11:24:19 modest-gator systemd[1]: Started Foreman jobs daemon - worker-2 on sidekiq.
+May 14 11:24:21 modest-gator dynflow-sidekiq@worker-2[1423]: 2020-05-14T11:24:21.448Z 1423 TID-gpcxyxoi7 INFO: GitLab reliable fetch activated!
+May 14 11:24:21 modest-gator dynflow-sidekiq@worker-2[1423]: 2020-05-14T11:24:21.449Z 1423 TID-gpcyculz7 INFO: Booting Sidekiq 5.2.8 with redis options {:id=>"Sidekiq-server-PID-1423", :url=>"redis://localhost:6379/0"}
+```
+
+### Scaling down
+Scaling down is just reverse process, let's say having `worker`, `worker-1` and `worker-2` was
+an overkill and we think we would be fine with just `worker` and `worker-1`.
+```
+root@modest-gator:/etc/foreman/dynflow# systemctl disable --now dynflow-sidekiq@worker-2
+Removed /etc/systemd/system/multi-user.target.wants/dynflow-sidekiq@worker-2.service.
+
+root@modest-gator:/etc/foreman/dynflow# rm /etc/foreman/dynflow/worker-2.yml
```

Comment:
I wonder why we don't actually ship 42 symlinks so users would not need to create or remove them and they could simply start and stop services. Overall I very much like this design, good work.

Anyway, I think you can leave a note that the file can be left there and won't hurt (I guess) until more instances are required.

Reply:
Btw 42 was just joke, 1-9 would be probably enough. We could even create a helper command: `foreman-maintain dynflow scale-up --by 3` :-)

Reply:
The original idea was to have the installer to this, but it somehow fell through the cracks. Not having to do this by hand would be nice